src/interfaces/coral_web/Dockerfile

```diff
@@ -36,5 +37,10 @@ CMD \
 
 
 # Production specifc tareget
-# FROM base AS prod
-# CMD npm run start
\ No newline at end of file
+FROM base AS prod
+CMD \
+  if [ -f yarn.lock ]; then yarn start; \
+  elif [ -f package-lock.json ]; then npm run start; \
+  elif [ -f pnpm-lock.yaml ]; then pnpm start; \
```

Comment:
Can we drop the support for all the other lockers?

Reply:
yes I want to, but will do it in a separate PR, because we have to fix the `readme.md` as well and other files

Reply:
https://linear.app/cohereai/issue/OS-2282/[fe]-remove-yarn-and-pnpm-references-in-toolkit